### PR TITLE
pull request feature design

### PR DESCRIPTION
**Pull request by touriasay**

### CODE REVIEW

**Comment by touriasay** [comment]
modif design création

File: src/main/java/view/IHMCreationLoserBracket.java [hunk 1]
```diff
@@ -12,20 +12,29 @@
...
+
 import java.util.ArrayList;
 
 import static utils.BusinessConstants.*;
 
 /**
  * IHM liée à la création d'un tournoi de type LooserBracket
  *
- * @author Carolane Pulval-Dady
+ * @author Carolane Pulval-Dady,Touria SAYAGH
```
Comment (Aaravoss) [inline src/main/java/view/IHMCreationLoserBracket.java]: @author Carolane Pulval-Dady
@author Touria SAYAGH

File: src/main/java/view/IHMCreationTournoi.java [hunk 2]
```diff
@@ -7,24 +7,34 @@
...
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
 import static utils.BusinessConstants.*;
 
 /**
  * IHM liée à la création d'un tournoi
  *
- * @author Carolane Pulval-Dady
+ * @author Carolane Pulval-Dady, Touria SAYAGH
```
Comment (Aaravoss) [inline src/main/java/view/IHMCreationTournoi.java]: @author Carolane Pulval-Dady
@author Touria SAYAGH

File: src/main/java/view/IHMCreationTournoiClassique.java [hunk 3]
```diff
@@ -12,20 +12,25 @@
...
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
 
 import static utils.BusinessConstants.*;
 
 /**
  * IHM liée à la création d'un tournoi Classique
  *
- * @author Carolane Pulval-Dady
+ * @author Carolane Pulval-Dady,Touria SAYAGH
```
Comment (Aaravoss) [inline src/main/java/view/IHMCreationTournoiClassique.java]: @author Carolane Pulval-Dady
@author Touria SAYAGH

File: src/main/java/view/IHMMenu.java [hunk 4]
```diff
@@ -4,20 +4,31 @@
...
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 
 import static utils.BusinessConstants.*;
 
 /**
  * IHM liée à l'affichage du menu principal
  *
- * @author Carolane Pulval-Dady
+ * @author Carolane Pulval-Dady, Touria SAYAGH
```
Comment (Aaravoss) [inline src/main/java/view/IHMMenu.java]: @author Carolane Pulval-Dady
@author Touria SAYAGH

**Review by Aaravoss — CHANGES_REQUESTED** [review]

**Review by Eilinia — APPROVED** [review]

**Review by touriasay — COMMENTED** [review]
changements faits

**Review by Aaravoss — APPROVED** [review]

**Pull request merged**